t/cli/test_admin.sh

```diff
@@ -112,6 +112,48 @@ if [ $count -eq 0 ]; then
     exit 1
 fi
 
+echo "
```

Comment:
The newer added tests don't check the behavior of `checked_admin_key = true`

Reply:
Thx. Fixed.

Reply:
Err... Even reverting the feature, the newer added tests can still pass, as they don't check the behavior of `checked_admin_key = true`.

Reply:
Ohh, I got what you mean, now all the cases are `checked_admin_key = true`, should make it `checked_admin_key = false`, add more test cases later.

Reply:
Added case to cover it. Plz take a view.

Reply:
Case as below I think it's ok.

The detail here, `yaml_conf.apisix.enable_admin` to `true`, `yaml_conf.apisix.allow_admin` not empty, so it through into foreach element to check `loopback` or not, but value `all` still stays `checked_admin_key` to `false`. 

So this case satisfies `if yaml_conf.apisix.enable_admin and not checked_admin_key then` and walks inside.

Am I missing other? Please point it out, thanks very much.

```shell
echo "
apisix:
    enable_admin: true
    allow_admin:
        - all
" > conf/config.yaml
```

Reply:
The tests can be passed even without the new feature.

Reply:
Added case value `10.0.0.1`.

The branch `if yaml_conf.apisix.enable_admin and not checked_admin_key then` seems be out of range changed feature from `if allow_ip == "127.0.0.0/24" then` to `if _ip and _ip:is_loopback() then`.

I think the new test case should cover `if _ip and _ip:is_loopback() then`, like the value `all` `10.0.0.1` `127.0.0.9` `127.0.0.1/8` `::1` `::1/128`, these cases are `non-IP address` `non-loopback` `IPv4 loopback address` `IPv4 loopback CIDR` `IPv6 loopback address` `IPv6 loopback CIDR`.

Maybe I'm confused now.